#! /bin/sh
patch -p1 -l -f $* < $0
exit $?

From 643575f38f545456afc456e1045b914d3d06bbb8 Mon Sep 17 00:00:00 2001
From: Andre Heinecke <aheinecke@intevation.de>
Date: Wed, 29 Jun 2016 17:17:48 +0200
Subject: [PATCH 2/2] Ignore sent S/MIME Mails if S/MIME is disabled

* src/mapihelp.cpp (mapi_change_message_class): Add check for
S/MIME disabled.

--
Outlook sets sent Mails to IPM.Note.SMIME (without suffix). When
S/MIME is disabled we need to ignore these messages. Our own
messages that are marked as S/MIME messages on send even if they
are PGP Messages have the override message class set so they are not
affected by this.

GnuPG-Bug-Id: 2319
---
 src/mapihelp.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/mapihelp.cpp b/src/mapihelp.cpp
index 7d11364..e916209 100644
--- a/src/mapihelp.cpp
+++ b/src/mapihelp.cpp
@@ -1226,7 +1226,7 @@ mapi_change_message_class (LPMESSAGE message, int sync_override)
         {
           newvalue = change_message_class_ipm_note (message);
         }
-      else if (!strcmp (s, "IPM.Note.SMIME"))
+      else if (opt.enable_smime && !strcmp (s, "IPM.Note.SMIME"))
         {
           newvalue = change_message_class_ipm_note_smime (message);
         }
--
2.1.4
